vignettes/GLM_intro_course.Rmd

```diff
@@ -2461,24 +2461,27 @@ testZeroInflation(r, plot = TRUE)
 
 ## Hurdle fit of zero-augmented data `r .emo("practice")`
 
-You can fit hurdle models with `pscl::hurdle()`:
+You can fit hurdle models with `glmmTMB::glmmTMB()`.  **{glmmTMB}** is an extremely flexible package that is relatively similar to **{spaMM}** (more details on differences between them later).
+
 ```{r hurdle, message = FALSE}
-library(pscl)
-fit_Zhurd1 <- hurdle(eggs ~ humans_eaten | humans_eaten, dist = "poisson", zero.dist = "binomial",
-                     data = AliensZ)
-fit_Zhurd2 <- hurdle(eggs ~ humans_eaten | 1, dist = "poisson", zero.dist = "binomial",
-                     data = AliensZ)
+library(glmmTMB)
+fit_Zhurd1 <- glmmTMB(eggs ~ humans_eaten, ziformula = ~humans_eaten,
+                      family = glmmTMB::truncated_poisson(link = "log"), data = AliensZ)
+fit_Zhurd2 <- glmmTMB(eggs ~ humans_eaten, ziformula = ~1,
+                      family = glmmTMB::truncated_poisson(link = "log"), data = AliensZ)
 lmtest::lrtest(fit_Zhurd1, fit_Zhurd2)
```

Comment:
```suggestion
library(glmmTMB)
fit_Zhurd1 <- glmmTMB(eggs ~ humans_eaten, ziformula = ~humans_eaten,
                      family = truncated_poisson(link = "log"), data = AliensZ)
fit_Zhurd2 <- glmmTMB(eggs ~ humans_eaten, ziformula = ~1,
                      family = truncated_poisson(link = "log"), data = AliensZ)
lmtest::lrtest(fit_Zhurd1, fit_Zhurd2)
```

Reply:
No issues, just thought it could be good to make it that it's not a 'standard' family from {stats}. Happy to remove though.